### ISSUE TITLE
the jupyter desktop couldn't be open because of novnc js was forbidden because of 403 http error

### ISSUE THREAD

**Issue by etherflow-cn** [jupyterhub/jupyter-remote-desktop-proxy#116]
### Bug description
after build the docker images, and run it with the description, try to open the desktop, with the connection pending. 
check the chrome and backend, with the following errors: 

 To access the server, open this file in a browser:
        file:///home/jovyan/.local/share/jupyter/runtime/jpserver-7-open.html
    Or copy and paste one of these URLs:
        http://eff3f15826c5:8888/lab?token=f0a61a27094bfa1ac86812c1473c670eb892f0e126017307
        http://127.0.0.1:8888/lab?token=f0a61a27094bfa1ac86812c1473c670eb892f0e126017307
[I 2024-06-21 06:10:31.654 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
[I 2024-06-21 06:10:47.124 LabApp] Build is up to date
[I 2024-06-21 06:10:48.834 ServerApp] 302 GET /desktop (fb74f02a2d9a45eeae9dfea049dd32a1@172.17.0.1) 0.78ms
[W 2024-06-21 06:11:12.849 ServerApp] Blocking request with no referer
[W 2024-06-21 06:11:12.849 ServerApp] 403 GET /desktop/static/dist/viewer.js.map (172.17.0.1): Blocking request from unknown origin
[W 2024-06-21 06:11:12.864 ServerApp] 403 GET /desktop/static/dist/viewer.js.map (fb74f02a2d9a45eeae9dfea049dd32a1@172.17.0.1) 15.10ms referer=None
[W 2024-06-21 06:11:12.866 ServerApp] Blocking request with no referer
[W 2024-06-21 06:11:12.866 ServerApp] 403 GET /desktop/static/dist/index.css.map (172.17.0.1): Blocking request from unknown origin
[W 2024-06-21 06:11:12.867 ServerApp] 403 GET /desktop/static/dist/index.css.map (fb74f02a2d9a45eeae9dfea049dd32a1@172.17.0.1) 1.93ms referer=None
<img width="1677" alt="image" src="https://github.com/jupyterhub/jupyter-remote-desktop-proxy/assets/69624847/3f4455d9-6bbd-416a-b7da-f94025760e6c">

### How to reproduce

build the docker image with the repo, 
run the docker with the command line: 
docker run --rm --security-opt seccomp=unconfined -p 8888:8888 xxxximage

#### Expected behaviour
could open the desktop.

### Your personal set up
try to install the docker image  on k3s, but has the same problem.
 - OS:
mac os 
 - Version(s):
 

<details><summary>Full environment</summary>

```
# paste output of `pip freeze` or `conda list` here
```
</details>

<details><summary>Configuration</summary>

```python
# jupyterhub_config.py
```
</details>

<details><summary>Logs</summary>

```
# paste relevant logs here, if any
```
</details>

**Comment by manics** [comment]
This is due to https://github.com/jupyterhub/jupyter-remote-desktop-proxy/issues/117

**Comment by consideRatio** [comment]
Resolved by pinning novnc 1.4 for now, closed in favor of #117
